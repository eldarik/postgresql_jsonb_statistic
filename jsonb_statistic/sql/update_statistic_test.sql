CREATE EXTENSION jsonb_statistic;

SELECT update_statistic('tablename', 'attname', 'key_path', 'value', 'text');

SELECT * from jsonb_statistics;

SELECT update_statistic('tablename', 'attname', 'key_path', 'value', 'text');

SELECT * from jsonb_statistics;

DROP EXTENSION jsonb_statistic;
