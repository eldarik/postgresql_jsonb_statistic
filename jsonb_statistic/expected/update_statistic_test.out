CREATE EXTENSION jsonb_statistic;
SELECT update_statistic('tablename', 'attname', 'key_path', 'value', 'text');
 update_statistic 
------------------
 
(1 row)

SELECT * from jsonb_statistics;
 tablename | attname | key_path | value | value_type | values_count 
-----------+---------+----------+-------+------------+--------------
 tablename | attname | key_path | value | text       |            1
(1 row)

SELECT update_statistic('tablename', 'attname', 'key_path', 'value', 'text');
 update_statistic 
------------------
 
(1 row)

SELECT * from jsonb_statistics;
 tablename | attname | key_path | value | value_type | values_count 
-----------+---------+----------+-------+------------+--------------
 tablename | attname | key_path | value | text       |            2
(1 row)

DROP EXTENSION jsonb_statistic;
